### DISCUSSION TITLE
Icon color in badge component

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2500] [Help: Figma] [answered]
Hey DSYS team!

I have been using the badge component in figma and I have noticed a recent change that the color of the badge changed from following with the background color to a fixed gray color which conflicts with the examples in the Paste website : https://paste.twilio.design/components/badge. I am wondering if that's a bug in the figma component or you recommend to use the same grey color for icons inside all badges. Thanks. 

Figma component screenshots attached 
<img width="327" alt="image" src="https://user-images.githubusercontent.com/82524550/176277636-712cfb6f-a868-4912-ab93-240966da59cf.png">

**Comment by TheSisb** [comment]
Hey @frankfzhao,

We're not seeing the same issue in the component library... can you share your Figma file with us?

**Reply by frankfzhao** [reply]
Hey Shadi, here is the link to my figma file: https://www.figma.com/file/PpH8AyERHPgIXT8yUu2BT8/Content-API---Editor-V2?node-id=425%3A849851. Super weird, this situation only happens in this specific file. All other files are in correct color

**Reply by kate-porter** [reply]
Thanks @frankfzhao! I just requested edit access to that file—if you don't mind giving me edit access, that'll help us troubleshoot the issue.

**Reply by loreina** [reply]
Hi @frankfzhao, I wasn't able to reproduce this in another file. It seems that the issue might be isolated in the file you shared, which is similar to another bug we've encountered before. I shared the bug with Figma support and will get back to you when I hear from them. Thanks for being patient, hopefully we'll get this sorted soon.

**Comment by loreina** [comment, marked as answer]
Hi @frankfzhao, Figma Support got back to us with a fix.

Can you open the Quick Actions menu (Command + /) and run "Repair component connections"? It'll take a few seconds to run, but it should fix the issue with the styles not being there for the Instances. I tested this on a duplicate of the file and it worked there!

You can also revert the repair (Command + Z) if you want to go back to its current state. Thanks for bringing this to our attention!

**Reply by frankfzhao** [reply]
Hey @loreina & @katieporter thanks so much for the help! The issue is fixed in my file.
